staging/src/k8s.io/client-go/rest/connection_test.go

```diff
@@ -43,6 +43,7 @@ type tcpLB struct {
 }
 
 func (lb *tcpLB) handleConnection(in net.Conn, stopCh chan struct{}) {
+	errs := make(chan error)
```

Comment:
this is not surfacing the error to the parent goroutine, and will block the send below. I'm afraid a proper fix is more complex than the rest of the PR is: 
- the parent goroutine needs to receive from the channel, to properly call `t.Fatalf`
- we need to be careful about when the goroutines return. If a parent goroutine returns before a child goroutine finishes / sends to the channel, the error will be ignored.
I wonder if we should leave this staticcheck error as it is and fix it in a separate PR.

Reply:
Okay , I will leave this in another PR. But, if that, I don't konw `vendor/k8s.io/client-go/rest` and `vendor/k8s.io/client-go/rest/watch` should be removed from `hack/.staticcheck_failures`?

Reply:
sadly yes.. but if you could paste the new static check result and show that now it only contains this one failure, I think that's a great achievement.